templates/terraform/processing-project/buckets.tf

```
@@ -95,3 +95,20 @@ resource google_storage_bucket_iam_member staging_account_iam_reader {
   role = "roles/storage.objectViewer"
   member = "serviceAccount:${var.jade_repo_email}"
 }
+
+# Bucket for long term Argo logs storage, currently want no "delete after N days" rule.
+resource google_storage_bucket argo_logs_storage {
```

```suggestion
resource google_storage_bucket argo_archive {
```
For consistency with suggested bucket name